frontend/src/components/sign/components/sign-in-form/sign-in-form.tsx

```diff
@@ -19,9 +19,14 @@ import { AuthActionCreator } from 'store/slices';
 import styles from './signin.module.scss';
 
 const SignInForm: React.FC = () => {
+  const preloadedValues = {
+    [LoginPayloadKey.EMAIL]: 'user@gmail.com',
+    [LoginPayloadKey.PASSWORD]: 'password',
+  };
```

Comment:
```suggestion
  const SIGN_IN_DEFAULT_VALUES = {
    [LoginPayloadKey.EMAIL]: '',
    [LoginPayloadKey.PASSWORD]: '',
  };
```
Move it to the
frontend/src/components/sign/components/common/constants
(don't forget to create a proxy import)